src/pages/search/project_search/ProjectFacetsFilter.tsx

```diff
@@ -34,13 +35,16 @@ export const ProjectFacetsFilter = ({ projectQuery }: ProjectFacetsFilterProps)
   const selectedParticipants = searchParams.get(ProjectSearchParameter.ParticipantFacet)?.split(',') ?? [];
   const selectedFundingSources = searchParams.get(ProjectSearchParameter.FundingSourceFacet)?.split(',') ?? [];
 
-  const addFacetFilter = (id: string) => {
-    const searchParameters = new URL(id).searchParams;
-    const newSearchParams = new URLSearchParams(
-      `${SearchParam.Type}=${currentSearchType}&${searchParameters.toString()}`
-    );
-    newSearchParams.set(SearchParam.Page, '1');
-    history.push({ search: newSearchParams.toString() });
+  const addFacetFilter = (param: string, key: string) => {
+    const currentValues = searchParams.get(param)?.split(',') ?? [];
```

Comment:
Er dette virkelig sånn det må gjøres? Hva hvis noen søker på noe med komma i seg?

Reply:
Her i kontekst av fasetter så kan man bare søke på IDer, så der vet vi at vi ikke har noen komma. I tilfeller hvor vi kan ha komma tror jeg løsningen må være å bruke `?param=xxx&param=yyy` i stedet for `?param=xxx,yyy`. Men det blir en del ekstra logikk for en sånn løsning da, siden man ikke lenger bare kan slå opp `param` og håndtere verdien som er der. I stedet må man hente alle params med det navnet, så vurdere hvilken param som skal endres/fjernes
Kommaseparering er/blir et problem for noen ting på resultatsøk da, feks om bruker vil søke på flere termer i tittel eller sammendrag, hvor en del av termen kan ha komma. Det håndterer vi ikke på noen god måte per i dag 🙃